Build/Module.xml

```diff
@@ -12,5 +12,9 @@
     <Package Uri="https-git://github.com/OutOfOrder/MonoKickstart.git" Folder="ThirdParty/Kickstart" GitRef="master" />
```

Comment:
Relax, I already know what's wrong with Mac pipeline tool, this will not fix it, but I needed this done so I can fix it in #4904